code/game/turfs/floor_types.dm

```diff
@@ -2306,9 +2306,6 @@
 /turf/open/floor/corsat/squareswood/north
 	dir = NORTH
 
-/turf/open/floor/corsat/tan
-	icon_state = "tan"
-
```

Comment:
Why are you deleting this?

Reply:
I don't remember doing this, don't know why i would.